forms-web-app/src/views/submit-appeal/grounds-appeal.njk

```diff
@@ -0,0 +1,151 @@
+              you should refer to them in your appeal statement and explain why they are relevant.</p>
+            <h2 class="govuk-heading-s">Responding to comments from interested parties</h2>
+            <p>There may have been comments (also called ‘representations’) from interested parties on the original
+              planning application. If you think these unreasonably influenced the local planning department to refuse
+              your planning application, refer to them here and explain why you do not think they are relevant.</p>
+            <p><strong>For example:</strong> interested parties commented on the planning application that the proposal
+              would affect the character of the local area, but other nearby properties have had similar proposals
+              approved.</p>
+          </div>
+        </details>
+        <p>Your statement cannot include any sensitive information.​</p>
+        <details class="govuk-details" data-module="govuk-details">
+          <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              What is sensitive information?
+            </span>
+          </summary>
+          <div class="govuk-details__text">
+            <h2 class="govuk-heading-s">Use of language and sensitive information</h2>
+            <p>You should not use racist, inflammatory or abusive language in your statement, or your appeal will be
+              rejected. You should not include sensitive information (also called ‘special category information’) in
+              your statement. If you do, you will be asked to resubmit and this will delay your appeal (or invalidate
+              it, if you do not resubmit in time).</p>
+
+            <p>Sensitive information refers to:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>Representations from children</li>
+              <li>Information relating to children</li>
+              <li>Information relating to health</li>
+              <li>Information relating to crime</li>
+              <li>Any information relating to an individual’s:
+                <ol type="i">
+                  <li>race</li>
+                  <li>ethnic origin</li>
+                  <li>politics</li>
+                  <li>religion</li>
+                  <li>trade union membership</li>
+                  <li>genetics</li>
+                  <li>physical characteristics</li>
+                  <li>sex life</li>
+                  <li>sexual orientation</li>
+                </ol>
+              </li>
+            </ul>
+          </div>
+        </details>
+        <div class="govuk-inset-text">File size should be no more than 1.5GB</div>
+        <div class="govuk-form-group">
+          <label class="govuk-heading-m" for="appeal-statement">
+            Upload a file
+          </label>
+          <input value="{{ data['appeal-statement'] }}" class="govuk-file-upload" id="appeal-statement"
```

Comment:
please could we use the gov uk component for this?

https://design-system.service.gov.uk/components/file-upload/

Reply:
Yes at the moment, it was just a dumb copy paste from the prototype, all the logics are in other US 
https://pins-ds.atlassian.net/browse/UCD-832
for example